src/main/java/com/fryrank/model/Address.java

```diff
@@ -0,0 +1,21 @@
+package com.fryrank.model;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Data
+public class Address {
+    @NonNull
+    private final String streetNumberAndName;
```

Comment:
I think it's better to separate street number and name. Integer for street number, String for name